R/derive_var_base.R

```diff
@@ -34,19 +34,19 @@
 #'
 #' @examples
 #' dataset <- tibble::tribble(
-#'   ~STUDYID, ~USUBJID, ~PARAMCD,  ~AVAL,  ~AVALC,   ~AVISIT,    ~ABLFL,
-#'   "TEST01", "PAT01",  "PARAM01", 10.12,  NA,       "Baseline", "Y",
-#'   "TEST01", "PAT01",  "PARAM01",  9.7,   NA,       "Day 7",    "N",
-#'   "TEST01", "PAT01",  "PARAM01", 15.01,  NA,       "Day 14",   "N",
-#'   "TEST01", "PAT01",  "PARAM02",  8.35,  NA,       "Baseline", "Y",
-#'   "TEST01", "PAT01",  "PARAM02", NA,     NA,       "Day 7",    "N",
-#'   "TEST01", "PAT01",  "PARAM02",  8.35,  NA,       "Day 14",   "N",
-#'   "TEST01", "PAT01",  "PARAM03", NA,     "LOW",    "Baseline", "Y",
-#'   "TEST01", "PAT01",  "PARAM03", NA,     "LOW",    "Day 7",    "N",
-#'   "TEST01", "PAT01",  "PARAM03", NA,     "MEDIUM", "Day 14",   "N",
-#'   "TEST01", "PAT01",  "PARAM04", NA,     "HIGH",   "Baseline", "Y",
-#'   "TEST01", "PAT01",  "PARAM04", NA,     "HIGH",   "Day 7",    "N",
-#'   "TEST01", "PAT01",  "PARAM04", NA,     "MEDIUM", "Day 14",   "N"
+#'   ~STUDYID, ~USUBJID, ~PARAMCD, ~AVAL, ~AVALC, ~AVISIT, ~ABLFL,
+#'   "TEST01", "PAT01", "PARAM01", 10.12, NA, "Baseline", "Y",
+#'   "TEST01", "PAT01", "PARAM01", 9.7, NA, "Day 7", "N",
+#'   "TEST01", "PAT01", "PARAM01", 15.01, NA, "Day 14", "N",
+#'   "TEST01", "PAT01", "PARAM02", 8.35, NA, "Baseline", "Y",
+#'   "TEST01", "PAT01", "PARAM02", NA, NA, "Day 7", "N",
+#'   "TEST01", "PAT01", "PARAM02", 8.35, NA, "Day 14", "N",
+#'   "TEST01", "PAT01", "PARAM03", NA, "LOW", "Baseline", "Y",
+#'   "TEST01", "PAT01", "PARAM03", NA, "LOW", "Day 7", "N",
+#'   "TEST01", "PAT01", "PARAM03", NA, "MEDIUM", "Day 14", "N",
+#'   "TEST01", "PAT01", "PARAM04", NA, "HIGH", "Baseline", "Y",
+#'   "TEST01", "PAT01", "PARAM04", NA, "HIGH", "Day 7", "N",
+#'   "TEST01", "PAT01", "PARAM04", NA, "MEDIUM", "Day 14", "N"
```

Comment:
To avoid that `style_file()` removes the alignment you have to use at least version 1.5.1 of styler and all columns must be left or right aligned. Here the `AVAL` column is not aligned. I would right align it and add trailing zeros such that the decimal points are in line.